saleor/product/utils/__init__.py

```diff
@@ -12,6 +13,21 @@
 from .availability import products_with_availability
 
 
+def get_default_automatic_fulfillment() -> bool:
```

Comment:
Would it make sense to have a single function instead of those three, that would return a dictionary of all digital download defaults? You would have to fetch `site` only once. Actually, one thing to consider here is that we have a middleware that stores the current site in `request.site` so that you don't have to fetch it every time, maybe we could take advantage of that somehow?